### PR TITLE
add friendly support for our celsius readers

### PR DESCRIPTION
**Pull request by MCer4294967296**
The title says everything.

### CODE REVIEW

File: ircbot/plugin/weather.py [hunk 1]
```diff
@@ -6,16 +6,17 @@
 
 
 def register(bot):
-    bot.listen(r'^(?:weather|hot|cold) ?(.*)$', weather, require_mention=True)
+    bot.listen(r'^(?:weather|hot|cold)(-c)? ?(.*)$', weather, require_mention=True)
```
Comment (dkess) [inline ircbot/plugin/weather.py]: Is this supposed to be in the style of command line flags? If so there should be a space here.

Comment (MCer4294967296) [reply ircbot/plugin/weather.py]: So I previously wanted to do it flag-style, but I was not sure about the spacing here (that's a question mark after the space, minor concerns). Therefore I later decided that it's acceptable to put -c right after the prompt, so no space.

Comment (dkess) [reply ircbot/plugin/weather.py]: You should be able to handle spacing by using the whitespace wildcard (`\s+`)

Comment (MCer4294967296) [reply ircbot/plugin/weather.py]: Then what do you think about the ` ?` after that? Is it also better to use `\s+`?

Comment (dkess) [reply ircbot/plugin/weather.py]: `r'^(?:weather|hot|cold)(\s+-c)?(\s+.*)$'` should be fine. I assume the weather API will strip trailing and leading whitespace from the location.

**Comment by MCer4294967296** [comment]
Okay there actually are some more work in terms of color and the unit that appears in the message.

**Comment by MCer4294967296** [comment]
I am having a hard time deciding how to describe the functionality of `-c` on the help page. Some help plz?

**Comment by dkess** [comment]
The following options are what I came up with:

* Add a sentence to the help text explaining that `-c` uses celsius
* Make it a different command entirely and document it as such, in a manner similar to widetext (where each degree of widening is in its own bullet point on the help page)

**Pull request merged**